docs/knowledge-base/posts/routing-l2-networks.md

```diff
@@ -0,0 +1,111 @@
+    Network Troubleshooting : [VergeOS Product Guide - Network Troubleshooting](/product-guide/networks/net-troubleshooting/)
+
+### Demo Scenario
+In our sample scenario below, we'll be routing the 192.168.2.0/24 address space via the VergeOS External interface to a layer2 network named "l2demo". In this scenario, any VM attached to l2demo with a static IP (in the correct address space) or with DHCP enabled would be able to reach the internet, as would any physical devices outside of Verge, via the L2 interface. Inbound traffic could be allowed and controlled via further rules. Outbound traffic could be restricted via further rules as well, paying attention to rule order.
+
+#### Demo Scenario Details
+- IP Block to Route: 192.168.2.0/24
+- Verge External IP: 10.1.1.2
+- Verge External L2 Network: l2demo
+- L2 Network ID : 1010
+- Verge Upstream Gateway: 10.1.1.1
+- Verge Physical Interface: phys1
+
+#### Create the L2 network
+	In Home > Networks > Externals
+1. Click New External
+2. Name: l2demo
+3. Description: Optional
+4. Layer 2 Type: vLan
+5. Layer 2 ID: 1010
+6. IP Address Type: None
+7. Interface Network: phys1
+8. Click Submit to save.
+9. Click "Power On" on the network page after saving. 
+   - Ensure the network powers up without errors in the Log box
+    
+#### Create IP block on External and Assign it
+1. Click "Network Blocks" on the External network page
+2.  Click "New"
+3.  Network Block: 192.168.2.0/24
+4.  Owner Type: Network
+5.  Owner: l2demo
+6.  Click Submit
+You will be returned to the Network Blocks page, now showing your new block and the Owner it is assigned to.
+    
+#### Assign IP info to L2 network
+1. Return to the l2demo network
+2. Click "Rules"
+  - Confirm Firewall rules are awaiting application. Check to confirm there is now an Outgoing route; the new rule. We'll apply it later.
+  - Click Cancel to exit back to the network page
+3. Click Edit to assign IP info to the network.
+4. IP Address Type: Static
+5. IP Address: 192.168.2.1
+6. Network Address 192.168.2.0/24
+7. DNS: Simple
+8. DNS Server: 10.1.1.1
+9. DHCP: enable
+10. Dynamic DHCP: enable
+11. DHCP Start Address: 192.168.2.200
+12. DHCP Stop Address: 192.168.2.254
+13. Click Submit to save
+
+At this point you have created everything you should need, but it's still pending application and a network restart to add Layer2 to l2demo.
+- Return to your Networks Dashboard and click All Networks
+- Note the External and l2demo networks marked as "Needs FW Apply" and l2demo "Needs Restart"
+- Restart the l2demo network by selecting it and clicking "Restart".
+  - Accept the offer to apply firewall rules on restart and click "yes" to confirm.
+- Apply FW Rules on External by selecting it and clicking "Apply Rules".
+  - Click "yes" to confirm
+- OUTSIDE OF VERGE: Set a route on 10.1.1.1 to send 192.168.2.0/24 via 10.1.1.2 and set any other required policies for traffic on that device.
```

Comment:
Just a nit-pick here, but it might be better to say on the upstream router instead of Outside of Verge